src/index.ts

```diff
@@ -209,13 +209,26 @@ function main() {
                     return ans;
                 })
                 .then((ans: Answers) => {
-                    answers.adminPassword = ans.pwdFirstAttempt;
-                    answers.sshFilePath = ans.sshFilePath;
-                    deployUI.start('Registering application in the Azure Active Directory');
-                    return createServicePrincipal(answers.azureWebsiteName, answers.subscriptionId, cachedAuthResponse.options);
+                    if (program.sku.toLowerCase() === solutionSkus[solutionSkus.local]) {
+                        return Promise.resolve({
```

Comment:
Can you add a comment here as to why we are  resolving promise with empty values. Like "For local deployment we don't need to create Application in AAD hence skipping the creation by resolving empty promise"

Reply:
added comment